data/academics.ts

```diff
@@ -1,804 +1,1311 @@
+  },
+  {
+    "name": "2A",
+    "value": ["CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHE 102","CHEM 102","CS 241","CS 241","CS 241","CS 241E","CS 241E","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ECE 222","ENGL 109","SE 212","SE 212","SPCOM 223","SPCOM 223","SPCOM 223","SPCOMM 223","STAT 206","STAT 206","STAT 206"]
+  },
+  {
+    "name": "2B",
+    "value": ["CS 240","CS 247","CS 247","CS 247","CS 247","EARTH 121","EARTH 121","EARTH 121","ECON 101","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MATH 239","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","MSCI 261","PSYCH 256","SCI 238","SCI 238","STV 202"]
+  },
+  {
+    "name": "3A",
+    "value": ["CS 341","CS 341","CS 341","CS 349","CS 349","CS 349","CS 459","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MATH 213","MSCI 442","PHYS 234","SE 350","SE 350","SE 350","SE 350","SE 350","SE 350","SE 350","SE 350","SE 350","SE 350","SE 350","SE 465","SE 465","SE 465","SE 465","SE 465","SE 465","SE 465","SE 465","SE 465","SPCOM 228"]
+  },
+  {
+    "name": "3B",
+    "value": ["CS 343","CS 343","CS 343","CS 343","CS 343","CS 343","CS 343","CS 343","CS 348","CS 348","CS 348","CS 348","CS 348","CS 348","CS 348","CS 380","ECON 101","SE 348","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 380","SE 390","SE 464","SE 464","SE 464","SE 464","SE 464","SE 464","SE 464","SE 464","SE 464","SE 464","SE 465"]
+  },
+  {
+    "name": "4A",
+    "value": ["CS 480","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 358","ECE 454","ECE 457A","ECE 457A","ECE 457A","MSCI 442","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 463","SE 490","SE 490","SE 490","SPCOM 325","STV 100"]
+  }
+];
```

Comment:
@wesley-a-leung Could you please tally these up? Currently the bar graphs that we have do not support auto tally so we need to provide { name: "XYZ", count: 10 } manually.

For all the numeric entries like housing costs, we can just use a histogram so no need to tally.

Reply:
Can you provide an example of the full format? Let's take `OVERLOAD ` for example.

Reply:
Ah, since `OVERLOAD` is per term, I'll probably have to make a separate bar graph for each term (i.e. 1 for 1A, 1 for 1B and so on) - lmk if this makes sense.
```
let OVERLOAD =  {
TERM_1A: [ { name: "yes", value: 1}, {name: "no", value: 53} ],
TERM_1B: [ ... ],
...
TERM_4B: [ ... ],
};
```

Reply:
Ok, I can work on this. Should be done by end of tomorrow.

Reply:
No rush! ty!

Reply:
@RobbieZhuang I've aggregated the data. Let me know if there's anything else you want changed.